performance/src/main/java/uk/gov/gchq/palisade/example/perf/config/PerformanceConfiguration.java

```diff
@@ -1,158 +1,158 @@
-
-import java.util.List;
-
-import static java.util.Objects.requireNonNull;
-
-public class PerformanceConfiguration {
-    private String action = "run";
-    private String directory;
-    private int small;
-    private int large;
-    private int manyUnique;
-    private int manyDuplicates;
-    private int dryRuns;
-    private int liveRuns;
-    private List<String> skipTests;
-    private String userId;
-    private String purpose;
-
-    @Generated
-    public String getAction() {
-        return action;
-    }
-
-    @Generated
-    public void setAction(final String action) {
-        requireNonNull(action);
-        this.action = action;
-    }
-
-    @Generated
-    public String getDirectory() {
-        return directory;
-    }
-
-    @Generated
-    public void setDirectory(final String directory) {
-        requireNonNull(directory);
-        this.directory = directory;
-    }
-
-    @Generated
-    public int getSmall() {
-        return small;
-    }
-
-    @Generated
-    public void setSmall(final int small) {
-        requireNonNull(small);
-        this.small = small;
-    }
-
-    @Generated
-    public int getLarge() {
-        return large;
-    }
-
-    @Generated
-    public void setLarge(final int large) {
-        requireNonNull(large);
-        this.large = large;
-    }
-
-    @Generated
-    public int getManyUnique() {
-        return manyUnique;
-    }
-
-    @Generated
-    public void setManyUnique(final int manyUnique) {
-        requireNonNull(manyUnique);
-        this.manyUnique = manyUnique;
-    }
-
-    @Generated
-    public int getManyDuplicates() {
-        return manyDuplicates;
-    }
-
-    @Generated
-    public void setManyDuplicates(final int manyDuplicates) {
-        requireNonNull(manyDuplicates);
-        this.manyDuplicates = manyDuplicates;
-    }
-
-    @Generated
-    public int getDryRuns() {
-        return dryRuns;
-    }
-
-    @Generated
-    public void setDryRuns(final int dryRuns) {
-        requireNonNull(dryRuns);
-        this.dryRuns = dryRuns;
-    }
-
-    @Generated
-    public int getLiveRuns() {
-        return liveRuns;
-    }
-
-    @Generated
-    public void setLiveRuns(final int liveRuns) {
-        requireNonNull(liveRuns);
-        this.liveRuns = liveRuns;
-    }
-
-    @Generated
-    public List<String> getSkipTests() {
-        return skipTests;
-    }
-
-    @Generated
-    public void setSkipTests(final List<String> skipTests) {
-        requireNonNull(skipTests);
-        this.skipTests = skipTests;
-    }
-
-    @Generated
-    public String getUserId() {
-        return userId;
-    }
-
-    @Generated
-    public void setUserId(final String userId) {
-        requireNonNull(userId);
-        this.userId = userId;
-    }
-
-    @Generated
-    public String getPurpose() {
-        return purpose;
-    }
-
-    @Generated
-    public void setPurpose(final String purpose) {
-        requireNonNull(purpose);
-        this.purpose = purpose;
-    }
-}
+/*
```

Comment:
whys this file been changed? i cant see any differences

Reply:
Reverted